frontend/src/scenes/events/EventsTable.js

```diff
@@ -190,10 +160,110 @@ function _EventsTable({ fixedFilters, filtersEnabled = true, pageKey }) {
+                                          filters={{ properties }}
+                                      />
+                                  )
+                              }
+                              return <Property value={event.properties[e]} />
+                          },
+                          ellipsis: true,
+                      }
+              )
+    let columns = [
+        {
+            title: `Event${eventFilter ? ` (${eventFilter})` : ''}`,
+            key: 'event',
+            fixed: 'left',
+            width: 100,
+            render: function renderEvent(item) {
+                if (!item.event) {
+                    return {
+                        children: item.date_break
+                            ? item.date_break
+                            : newEvents.length === 1
+                            ? `There is 1 new event. Click here to load it.`
+                            : `There are ${newEvents.length} new events. Click here to load them.`,
+                        props: {
+                            colSpan: 6,
+                            style: {
+                                cursor: 'pointer',
+                            },
+                        },
+                    }
+                }
+                let { event } = item
+                return eventToName(event)
+            },
+            filterIcon: function RenderFilterIcon() {
+                return (
+                    <SearchOutlined
+                        style={{ color: eventFilter && 'var(--primary)' }}
+                        data-attr="event-filter-trigger"
+                    />
+                )
+            },
+            filterDropdown: function RenderFilter({ confirm }) {
+                return (
+                    <div style={{ padding: '1rem' }}>
+                        <Button
+                            style={{ float: 'right', marginTop: -6, marginBottom: 8 }}
+                            onClick={() => {
+                                confirm()
+                                setEventFilter(false)
+                            }}
+                            type="primary"
+                            disabled={!eventFilter}
+                        >
+                            Reset
+                        </Button>
+                        Filter by event
+                        <EventName
+                            value={eventFilter}
+                            onChange={(value) => {
+                                confirm()
+                                setEventFilter(value)
+                            }}
+                        />
+                    </div>
+                )
+            },
+        },
+        ...columnsToRenderFromConfig,
+    ]
 
     return (
         <div className="events" data-attr="events-table">
             {filtersEnabled ? <PropertyFilters pageKey={'EventsTable'} /> : null}
+            <Button onClick={openColumnChooser} type="secondary">
```

Comment:
I think this would look better if it had the class `float-right`

Reply:
fixed

Reply:
Fixed in d9f2dac1acdda9fc27bde1751a81257e9877d387